braze/src/main/java/com/tealium/remotecommands/braze/BrazeConstants.java

```diff
@@ -46,7 +46,10 @@ private Config() {
 
         public static final String API_KEY = "api_key";
         public static final String FIREBASE_ENABLED = "firebase_enabled";
+        public static final String FIREBASE_NEW_TOKEN_ENABLED = "firebase_new_token_enabled";
         public static final String FIREBASE_SENDER_ID = "firebase_sender_id";
+        public static final String FIREBASE_FALLBACK_MESSAGE_SERVICE_ENABLED = "firebase_fallback_message_service_enabled";
+        public static final String FIREBASE_MESSAGING_SERVICE_CLASSPATH = "firebase_messaging_service_classpath";
```

Comment:
The naming format on these two constants differ slightly - but the method names they actually refer to, both use `fallback` and `messaging` - can we align them? or this keeping in line with ios in some way?